src/App.scss

```
@@ -13,3 +13,33 @@
   padding: 20px;
   border-left: 1px solid lightgray;
 }
+
+.warning__message {
+  color: rgb(255, 0, 0);
+}
+
+.accepted__message {
+  color: rgb(0, 126, 0);
+}
+
+.Add__film__title {
```

напиши правильно по БЕМу
block__element
block__element--modificator
```suggestion
.Add__film__title {
```